recipe/meta.yaml

```diff
@@ -6,54 +6,38 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/uproot-{{ version }}.tar.gz
```

Comment:
I understand that there's only one package now, but why should `uproot` be hard-coded?

Reply:
@jpivarski This is the behavior of `grayskull` now. I wondered the same thing, but I haven't bothered to ask on https://github.com/conda/grayskull.

Reply:
@jpivarski c.f. https://github.com/conda/grayskull/discussions/569

Reply:
I'm going to merge this in, but if there's some answer from the `grayskull` team that makes us think we should change from what they do then we can open a fix PR.